infrastructure/docker/Dockerfile-notebook

```diff
@@ -12,8 +12,16 @@ RUN pip install ipywidgets
 RUN cd ../
 RUN rm -r ./qs
 
-COPY --chown=$NB_UID:$NB_UID ./docs/tutorials/ ./serverless/tutorials/
-COPY --chown=$NB_UID:$NB_UID ./docs/guides/ ./serverless/guides/
-COPY --chown=$NB_UID:$NB_UID ./docs/getting_started/ ./serverless/getting_started/
+RUN git clone https://github.com/Qiskit-Extensions/circuit-knitting-toolbox.git ckt && \
```

Comment:
I think ckt team just release to Pypi. We might want to include this requirement in `/client/requirements.txt` 

https://pypi.org/project/circuit-knitting-toolbox/

Reply:
or we might not even needed anymore since users can do this now:
```python
Program(
  title="DO ckt",
  entrypoint="...",
  dependencies=["circuit-knitting-toolbox==0.1.0"]
)
```

Reply:
ah, did not know that (I just saw the tags, not the releases). that'll simplify things a good bit.

I'd still lean towards including it in the notebook dockerfile (and not to `requirements.txt`) as that would let non-Programs also have access to it